code/modules/antagonists/changeling/powers/tiny_prick.dm

```diff
@@ -41,7 +41,7 @@
 	user.hud_used.lingstingdisplay.invisibility = 101
 
 /datum/action/changeling/sting/can_sting(mob/user, mob/target)
-	if(!..() || !iscarbon(target) || !isturf(user.loc) || !AStar(user, target.loc, /turf/proc/Distance, cling.sting_range, simulated_only = 0))
+	if(!..() || !iscarbon(target) || !isturf(user.loc) || !get_path_to(user, target.loc, /turf/proc/Distance, cling.sting_range, simulated_only = 0))
```

Comment:
This use of `/turf/proc/Distance` causes a runtime.

![image](https://user-images.githubusercontent.com/25063394/192164474-f045c97f-73c8-4c8c-9cbb-da0073ee6ecc.png)

Reply:
Literally in the middle of my review this gets updated